snippets/dart_2_snippets.md

```diff
@@ -0,0 +1,53 @@
+## Available Dart 1 and Dart 2 Backwards Compatible Code Snippets
+
+Below is a list of all available Dart 2 builder-compatible over_react snippets and their<br>
+triggers for Webstorm/IntelliJ and VS Code. 
+
+| Trigger  | Content |
+| -------: | ------- |
+| `orStless`   | Dart 2 stateless component skeleton |
+| `orCStless`   | Dart 2 commented stateless component skeleton |
+| `orAbsStless`  | Dart 2 abstract stateless component skeleton |
+| `orCAbsStless`  | Dart 2 commented abstract stateless component skeleton |
+| `orStful`   | Dart 2 stateful component skeleton |
+| `orCStful`   | Dart 2 commented stateful component skeleton |
+| `orAbsStful`  | Dart 2 abstract stateful component skeleton |
+| `orCAbsStful`  | Dart 2 commented abstract stateful component skeleton |
+| `orPropsMixin`  | Dart 2 prop mixin skeleton |
+| `orStateMixin`  | Dart 2 state mixin skeleton |
+
+## WebStorm and IntelliJ Snippets
+
+### Installation Instructions
+
+<img src="webstorm_intelliJ.gif" alt="How to install and use snippets in WebStorm and IntelliJ"/><br>
+
+1. Copy the command below and run it in your terminal.
+   
+   ```curl https://raw.githubusercontent.com/Workiva/over_react/278f66475bf0d9b892936dccff6e2aef88c14c1e/snippets/snippets.xml | pbcopy``` 
```

Comment:
Making this multiline will make it look much nicer:
```suggestion
	 ```
   curl https://raw.githubusercontent.com/Workiva/over_react/278f66475bf0d9b892936dccff6e2aef88c14c1e/snippets/snippets.xml | pbcopy
	``` 
```

----
### For example:
<img width="918" alt="screen shot 2018-12-21 at 12 28 41 pm" src="https://user-images.githubusercontent.com/12497362/50360065-04d80580-051c-11e9-8dd3-d3665221ccfd.png">